assets/css/styles.scss

```diff
@@ -7,6 +7,33 @@
   overflow: hidden;
 }
 
+nav a[aria-active=true]:link {
+  background: transparent;
+  // XXX this isn't turning links black for me...
+  color: #000;
+  font-weight: bold;
+}
+
+// add the nav active indicator to targeted section titles
+section:target > h1,
```

Comment:
Will this have an effect on all `section`s and `h*`s? What if theres one in the footer or somewhere not in the main content?